torch/utils/data/graph.py

```diff
@@ -4,11 +4,13 @@
 from torch.utils.data import IterDataPipe, MapDataPipe
 from torch.utils.data._utils.serialization import DILL_AVAILABLE
 
-from typing import Any, Dict, Set, Tuple, Type, Union
+from typing import Dict, List, Set, Tuple, Type, Union
 
-__all__ = ["traverse", ]
+__all__ = ["DataPipe", "DataPipeGraph", "traverse"]
 
 DataPipe = Union[IterDataPipe, MapDataPipe]
+DataPipeGraph = Dict[int, Tuple[DataPipe, "DataPipeGraph"]]  # type: ignore[misc]
```

Comment:
Do their `__module__` need to change or they need to be added to `__init__` to fix the following CI error?
https://github.com/pytorch/pytorch/runs/7114135518?check_suite_focus=true